### PR TITLE
Rebased balancing

### PR DESCRIPTION
**Pull request by rishay-jain**

### CODE REVIEW

**Comment by ky28059** [comment]
@rishay-jain does this supersede #36?

**Comment by rishay-jain** [comment]
> @rishay-jain does this supersede #36?

Yes

**Comment by ky28059** [comment]
@e3l can we merge this or should we wait for more testing? This PR brings 2 working paths to main (balance no taxi and 1-piece bottom).

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -89,13 +90,13 @@ public class RobotContainer {
...
     private final SendableChooser<Command> autonChooser;
-    private final BalancerCommand balancerCommand;
+    private final DefaultBalancerCommand balancerCommand;
 
     /**
      * The container for the robot. Contains subsystems, OI devices, and commands.
      */
     public RobotContainer() {
-        driveController = new DualJoystickDriveController();
+        driveController = new XboxDriveController();
```
Comment (e3l) [inline src/main/java/frc/robot/RobotContainer.java:99]: ideally not this

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -186,10 +189,12 @@ private void configureButtonBindings() {
             }, swerveSubsystem));
         }
 
+        /*
         rollerSubsystem.setDefaultCommand(new RunCommand(() -> {
             double rollPower = mechController.getRightTriggerAxis() - mechController.getLeftTriggerAxis();
             rollerSubsystem.setRollPower(rollPower);
         }, rollerSubsystem));
+        */
```
Comment (e3l) [inline src/main/java/frc/robot/RobotContainer.java:197]: please not this

**Review by e3l — APPROVED** [review]
lgtm! only minor comments

**Pull request merged**